packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/components/Regenerate/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Button } from '@strapi/design-system/Button';
+import Refresh from '@strapi/icons/Refresh';
+import { ConfirmDialog } from '@strapi/helper-plugin';
+import { axiosInstance } from '../../../../../../../core/utils';
+
+const ButtonWithRightMargin = styled(Button)`
+  margin-right: ${({ theme }) => theme.spaces[2]};
+`;
```

Comment:
I've suggested below to use `Stack` instead of `Flex`, which avoids having these overwrites at all. Sorry, I didn't see that in my first review.

Reply:
done